### PR TITLE
LEAF-1719: Added sitemap page

### PR DESCRIPTION
**Pull request by abiola-adewuyi**
Added a new page for sitemap

### CODE REVIEW

File: LEAF_Request_Portal/templates/sitemap.tpl [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<link rel="stylesheet" href="../libs/css/leaf.css">
+<style>
+	.flex-container {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
```
Comment (dastan-rahimi) [inline LEAF_Request_Portal/templates/sitemap.tpl:8]: Move inline styles to leaf.css

Comment (abiola-adewuyi) [reply LEAF_Request_Portal/templates/sitemap.tpl]: Done.

Comment (dastan-rahimi) [reply LEAF_Request_Portal/templates/sitemap.tpl]: leaf.css is generated from sass under the libs folder. Make sure you add it there and run gulp watch to generate leaf.css

Comment (cameron-chilton) [reply LEAF_Request_Portal/templates/sitemap.tpl]: Abiola was having issues with installing gulp. The css issues will be fixed in the 1723: Update Sitemap Colors jira task.

**Pull request merged**